instrumentation/grpc-1.6/library/src/main/java/io/opentelemetry/instrumentation/grpc/v1_6/GrpcAttributesExtractor.java

```diff
@@ -5,18 +5,35 @@
 
 package io.opentelemetry.instrumentation.grpc.v1_6;
 
+import static io.opentelemetry.instrumentation.api.internal.AttributesExtractorUtil.internalSet;
+
 import io.grpc.Status;
+import io.opentelemetry.api.common.AttributeKey;
 import io.opentelemetry.api.common.AttributesBuilder;
 import io.opentelemetry.context.Context;
 import io.opentelemetry.instrumentation.api.instrumenter.AttributesExtractor;
 import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import java.util.List;
 import javax.annotation.Nullable;
 
 final class GrpcAttributesExtractor implements AttributesExtractor<GrpcRequest, Status> {
+  // TODO: This should be added to
+  //  io.opentelemetry.semconv.trace.attributes.SemanticAttributes when possible
```

Comment:
why is it currently blocked?

Reply:
@arbiv From what I gather, the constants are generated from a build script in https://github.com/open-telemetry/opentelemetry-specification 
Which means that it depends on a release.
I also consulted @samimusallam who works on the same implementation in JS and we both seem to agree that it should be solved in a future release.

Reply:
In case of "normal" attributes that would be true; but this (similar to the HTTP headers attributes) is sort of a "template" attribute name - there isn't one static attribute name, the spec defines a class of attributes that start with `rpc.request.metadata.`, which the generator does not currently handle. I think there's no need to keep that TODO in the code.

Reply:
I get what you say and I agree, removing it now